input/fsh/omrs-location.fsh

```diff
@@ -6,29 +8,44 @@ Description:  "An OpenMRS location"
 * identifier 1..1 
 * status 1..1 
 * status from LocationStatusVS
+* operationalStatus 0..0
```

Comment:
Make sure these fields represent what the fhir2 module maps here: https://github.com/openmrs/openmrs-module-fhir2/blob/d0b34f74c243b32669d86c9369f7b226a0ada2f9/api/src/main/java/org/openmrs/module/fhir2/api/translators/impl/LocationTranslatorImpl.java#L69